src/auth.rs

```diff
@@ -61,6 +62,24 @@ impl IntoResponse for ClaimsError {
 #[derive(Clone, Debug)]
 pub struct Claims(CoreUserInfoClaims);
 
+impl Claims {
+    pub fn has_starred(&self, repo_full_name: &str) -> anyhow::Result<bool> {
+        let mut subject_iter = self.subject().split('|');
```

Comment:
Looks like https://doc.rust-lang.org/std/primitive.str.html#method.split_once is what you're looking for.